.travis.yml

```diff
@@ -5,6 +5,9 @@ node_js:
 os:
   - linux
 
+script:
+  skip
```

Comment:
IIUC, this is to skip the scripts (`npm run fetch-readmes` and `npm test` I presume) until you get a green light right?

Reply:
At least that's what I'm aiming for. I'll revert this if it's not this behaviour.